scripts/git_page/wdlviz.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+"""
+Visualize a WDL workflow using miniwdl and graphviz
+
+This script is a modified version of the wdlviz.py script from the miniwdl repository:
+https://github.com/chanzuckerberg/miniwdl/blob/main/examples/wdlviz.py
+
+"""
+# black -l 100 wdlviz.py && pylint wdlviz.py
+import os
+import sys
+import argparse
+import tempfile
+import WDL
+import graphviz
+from urllib import request, parse
+
+
+def main(args=None):
+    # read command-line arguments
+    parser = argparse.ArgumentParser(
+        description="Visualize a WDL workflow using miniwdl and graphviz"
+    )
+    parser.add_argument(
+        "wdl", metavar="FILE", help="WDL workflow file or URL (- for standard input)"
+    )
+    parser.add_argument("--inputs", action="store_true", help="include input declarations")
+    parser.add_argument("--outputs", action="store_true", help="include output declarations")
+    parser.add_argument("--output_path", help="Path to output directory")
+    parser.add_argument(
+        "--no-quant-check",
+        dest="check_quant",
+        action="store_false",
+        help="relax static typechecking of optional types, and permit coercion of T to Array[T] (discouraged; for backwards compatibility with older WDL)",
+    )
+    parser.add_argument(
```

Comment:
support option of choice between `svg` and `png`

Reply:
done